### PR TITLE
Restrict manual verification to council+

### PR DESCRIPTION
**Pull request by alexkar598**
DNM pending council vote

### CODE REVIEW

**Comment by JamieD1** [comment]
Not sure why it needs to be council only.

File: data/discord.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     },
     "council": {
       "inherits": "head-developer",
-      "permissions": []
+      "permissions": ["userverify"]
     },
     "head-developer": {
       "inherits": "senior admin",
```
Comment (JamieD1) [inline data/discord.json:12]:
```suggestion
      "permissions": []
    },
    "head-developer": {
      "inherits": "senior admin",
      "permissions": ["userverify"]
```

**Comment by adamsong** [comment]
Didn't mean to make not WIP, hit the wrong button and can't find the button to undo on mobile

**Comment by alexkar598** [comment]
Merging due to https://forums.yogstation.net/threads/vote-871-merge-https-github-com-yogstation13-yogbot-pull-337.24793/

**Pull request merged**